rfc/5/index.md

```diff
@@ -9,495 +9,645 @@ responses/index
-axis types.
+Coordinate and spatial transformation are vitally important
+for neuro and bio-imaging and broader scientific imaging practices to enable:
+
+1. Reproducibility and Consistency:
+  Supporting spatial transformations explicitly in a file format ensures that
+  transformations are applied consistently across different platforms and applications.
+  This FAIR capability is a cornerstone of scientific research,
+  and having standardized formats and tools facilitates verification of results by independent researchers.
+2. Integration with Analysis Workflows: 
+  Having spatial transformations as a first-class citizen within file formats
+  allows for seamless integration with various image analysis workflows.
+  Registration transformations can be used in subsequent image analysis steps
+  without requiring additional conversion.
+3. Efficiency and Accuracy:
+  Storing transformations within the file format avoids
+  the need for re-sampling each time the data is processed.
+  This reduces sampling errors and preserves the accuracy of subsequent analyses.
+  Standardization enables on-demand transformation,
+  critical for the massive volumes collected by modern microscopy techniques.
+4. Flexibility in Analysis:
+  A file format that natively supports spatial transformations allows researchers to apply, modify,
+  or reverse transformations as needed for different analysis purposes.
+  This flexibility is critical for tasks such as longitudinal studies, multi-modal imaging,
+  and comparative analysis across different subjects or experimental conditions.
+
+Toward these goals, this RFC expands the set of transformations in the OME-Zarr spec
+covering many of the use cases requested in [this github issue](https://github.com/ome/ngff/issues/84).
+It also adds "coordinate systems" - named sets of "axes."
+Related the relationship of discrete arrays to physical coordinates
+and the interpretation and motivation for axis types.
 
 
 ## Proposal
 
-Below is a slightly abridged copy of the proposed changes to the specification (examples are omitted), the full set of changes
-including all examples are publicly available on the [github pull request](https://github.com/ome/ngff/pull/138).
+Below is a complete copy of the proposed changes including suggestions
+from reviewers and contributors of the previously associated [github pull request](https://github.com/ome/ngff/pull/138).
+The changes, if approved, shall be translated into bikeshed syntax and added to the ngff repository in a separate PR.
+This PR will then comprise complete json schemas when the RFC enters the SPEC phase (see RFC1).
 
 
 ### "coordinateSystems" metadata
 
-A "coordinate system" is a collection of "axes" / dimensions with a name. Every coordinate system:
-- MUST contain the field "name". The value MUST be a non-empty string that is unique among `coordinateSystem`s.
+A "coordinate system" is a collection of "axes" / dimensions with a name.
```

Comment:
the definition of the JSON object should probably be preceded by some description of the abstract thing the JSON object models. Otherwise, readers might not know what the point it is. For example, here I would state that a coordinate system defines a mapping from array indices of Zarr arrays to points in a domain with some scientific or application-specific meaning, and then move to the description of the JSON object.